tests/integration/components/code-mirror-test.js

```diff
@@ -0,0 +1,378 @@
+import { module, test, skip } from 'qunit';
+import { setupRenderingTest } from 'codecrafters-frontend/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+const SELECTOR_COMPONENT = '[data-test-code-mirror-component]';
```

Comment:
@VasylMarchuk is there a reason we can't use `ember-cli-page-object` here? That's what we use for other tests, and it seems like that'd be cleaner to use here too.